protobuf-maven-plugin/src/main/resources/META-INF/m2e/lifecycle-mapping-metadata.xml

```diff
@@ -0,0 +1,17 @@
+<lifecycleMappingMetadata>
```

Comment:
Just need to run `./mvnw license:format` to get the validation step to pass.

It might be worth adding a comment in here just to explain what the file is for too, just to help anyone working on this in the future (and to aid in my ignorance 🙂)